course3/homework-string/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -std=c++11")
```

Comment:
yeah I'm using gcc 4.8.4 (comes with ubuntu 14), I only discovered last night that the right flag for c++14 is -std=c++1y